app/utils/animation.ts

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Is there any reason as to why we are coding this and not using some library?

Reply:
I've been refactoring this pr based on some feedback from @ThomasRalee, he mentioned the same thing so this file will disappear in favor of a third party library.

Reply:
I refactored it to use css animation instead so no third party lib is needed.